site/reference/annotations/local-config/README.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "`local-config`"
+linkTitle: "local-config"
+type: docs
+description: >
+  Specify a resource to be skipped when applying.
+---
+
+<!--mdtogo:Short
+   Specify a resource to be skipped when applying.
+-->
+
+<!--mdtogo:Long-->
+
+The `config.kubernetes.io/local-config` annotation specifies a resource to be
+skipped when applying.
+
+These resources are generally used as input to kpt functions.
+
+Because local resources are not applied, they don't need to follow a resource
+schema known by the cluster. They just need to be a valid Kubernetes resource,
+with apiVersion, kind, metadata, and name.
+
+### Schema
+
+The annotation value accepts string values of `true` and `false`.
+
+Make sure to surround the value in quotes, otherwise it will be considered a
+YAML boolean (invalid annotation), not a string.
+
+<!--mdtogo-->
+
+### Behavior
+
+Resources with the `local-config` annotation set to `true` will not be applied
+to the cluster when using `kpt live apply`.
+
+### Example
+
+<!--mdtogo:Examples-->
+
+In this example, the ConfigMap `cm-a` is local and not applied.
```

Comment:
I think we usually use backticks around resource types, so this should be `ConfigMap`.